### PR TITLE
Bug: Fixed issue where Rotate Image & Set as Wallpaper were sometimes not displayed

### LINKED ISSUE #11810: Bug: IsExecutable for Set as Background action is broken
### Description

The IsExecutable state isn't updating properly for the Set as Background action. It looks like this issue affect the rotate image actions as well.

### Steps To Reproduce

1. Navigate to a folder with images as well as other file types
2. Click any file that isn't a supported image
3. Click an image file and see how the rotate toolbar buttons are disabled
4. See that the Set as Background toolbar button isn't even visible

### Requirements

- Fix IsExecutable state for the rotate buttons
- Fix visibility of the Set as Background button

### Files Version

v2.4.55

### Windows Version

Windows 11

### Log file

NA

### PR DESCRIPTION
**Pull request by mafra99**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11810

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Rotate & Select As Background Image are hidden if no item is selected
   2. Rotate & Select As Background Image are hidden if a non image file is selected
   3. Rotate & Select As Background Image are visible if an image is selected
   4. Set as desktop slideshow is visible if multiple images are selected

### CODE REVIEW

**Comment by yaira2** [comment]
> It might be better if the ViewModel would update the state itself so you don't have to call the method CheckAllFileExtensions in all three Actions but I couldn't find an easy way to implement this

Once we're finished with implementing the actions, we can look into improving the way they access the context as well as the code it uses to execute the logic.

**Comment by ferrariofilippo** [comment]
> It might be better if the ViewModel would update the state itself so you don't have to call the method `CheckAllFileExtensions` in all three Actions but I couldn't find an easy way to implement this

What about an abstract parent `BaseSetAsAction` so that we can share the code?

**Comment by ferrariofilippo** [comment]
> This does not change the fact that the method `CheckAllFileExtensions` is called in each action. But it is a good idea to avoid the duplicate code. I will create the abstract class later

Saving the result in a static property?

File: src/Files.App/Actions/Content/ImageEdition/RotateRightAction.cs [hunk 1]
```diff
@@ -1,65 +1,21 @@
...
-		public bool IsExecutable => IsContextPageTypeAdaptedToCommand()
-						&& (context.ShellPage?.SlimContentPage?.SelectedItemsPropertiesViewModel?.IsSelectedItemImage ?? false);
-
-		public RotateRightAction()
-		{
-			context.PropertyChanged += Context_PropertyChanged;
-		}
-
-		public async Task ExecuteAsync()
+		public RotateRightAction() : base()
```
Comment (d2dyno1) [inline src/Files.App/Actions/Content/ImageEdition/RotateRightAction.cs:17]: No need to explicitly call `base()`

**Review by yaira2 — DISMISSED** [review]
LGTM

**Comment by hishitetsu** [comment]
Please rename this too.
https://github.com/files-community/Files/blob/344a8901eda766e33d2669a56a7f9d19b55e7cdf/src/Files.App/Commands/CommandCodes.cs#L80-L82

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**